pepper-apis/src/main/java/org/broadinstitute/ddp/util/ActivityInstanceUtil.java

```diff
@@ -147,4 +161,19 @@ public static boolean computeCanDelete(boolean canDeleteInstance, Boolean canDel
         }
         return canDelete;
     }
+
+    public static void populateDefaultValues(Handle handle, long instanceId, long operatorId) {
+        var answerDao = new AnswerCachedDao(handle);
+        FormActivityDef activityDef = getActivityDef(handle, ActivityDefStore.getInstance(), instanceId);
+        List<QuestionDef> questions = QuestionUtil.collectQuestions(activityDef, questionDef ->
+                (questionDef instanceof PicklistQuestionDef) && !((PicklistQuestionDef) questionDef).getDefaultOptions().isEmpty());
+        for (QuestionDef question : questions) {
+            PicklistQuestionDef plQuestion = (PicklistQuestionDef) question;
+            var answer = new PicklistAnswer(null, plQuestion.getStableId(), null,
+                    plQuestion.getDefaultOptions().stream()
+                            .map(plOpt -> new SelectedPicklistOption(plOpt.getStableId()))
+                            .collect(Collectors.toList()));
+            answerDao.createAnswer(operatorId, instanceId, answer, plQuestion);
+        }
+    }
```

Comment:
Does this handle the case where an option has `isExclusive` set, or when there's a mismatch between the number of default options and any exclusivity or validation requirements?

Reply:
No, there is no any checks given the fact that it is a study design time feature in comparison to the runtime stuff like `isExclusive` and validations